OpenTabletDriver.UI.Library/Views/MainWindowView.axaml.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Avalonia.Controls;
+using Avalonia.Input;
+using Avalonia.Threading;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using CommunityToolkit.Mvvm.Messaging;
+using OpenTabletDriver.UI.Messages;
+using OpenTabletDriver.UI.Navigation;
+using OpenTabletDriver.UI.ViewModels;
+
+namespace OpenTabletDriver.UI.Views;
+
+public partial class MainWindowView : Window
+{
+    private readonly INavigator _navigator;
+    private readonly IDispatcher _dispatcher;
+
+    public MainWindowView()
+    {
+        InitializeComponent();
+        DataContext = Ioc.Default.GetRequiredService<MainWindowViewModel>();
+        _navigator = Ioc.Default.GetRequiredService<INavigatorFactory>().GetOrCreate(AppRoutes.MainHost);
+        _dispatcher = Ioc.Default.GetRequiredService<IDispatcher>();
+        var messenger = Ioc.Default.GetRequiredService<IMessenger>();
+
+        HookInputEvents();
+        this.BootstrapTransparency((MainWindowViewModel)DataContext!, _dispatcher);
+
+        // ensure we have window-level focus on startup
+        Activate();
+        Focus();
+
+        VIEW_SplitView.PaneOpened += (sender, e) =>
+        {
+            messenger.Send(new UILayoutChangedMessage(UILayoutChange.SidebarOpen));
+        };
+
+        VIEW_SplitView.PaneClosed += (sender, e) =>
+        {
+            messenger.Send(new UILayoutChangedMessage(UILayoutChange.SidebarHidden));
+        };
+    }
+
+    private void HookInputEvents()
+    {
+        PointerPressed += (sender, e) =>
+        {
+            if (e.Handled)
+                return;
+
+            e.Handled = true;
+            // is there a need to implement forward?
```

Comment:
Assuming this comment relates to back/forward buttons on side mouse, I would prefer forward being an option if Avalonia supports it